safekeeper/src/metrics.rs

```diff
@@ -573,18 +582,32 @@ impl TimelineCollector {
 
         let timelines_count = IntGauge::new(
             "safekeeper_timelines",
-            "Total number of timelines loaded in-memory",
+            "Total number of timelines loaded in-memory, including cancelled (deleted) ones.",
         )
         .unwrap();
         descs.extend(timelines_count.desc().into_iter().cloned());
 
+        let timelines_cancelled_count = IntGauge::new(
+            "safekeeper_timelines_cancelled",
+            "Number of cancelled timelines loaded in-memory",
+        )
+        .unwrap();
+        descs.extend(timelines_cancelled_count.desc().into_iter().cloned());
+
         let active_timelines_count = IntGauge::new(
             "safekeeper_active_timelines",
             "Total number of active timelines",
         )
         .unwrap();
         descs.extend(active_timelines_count.desc().into_iter().cloned());
 
+        let active_timelines_set_size = IntGauge::new(
```

Comment:
I'm not sure if we should add this metric. The downside is that having several different metrics for a similar value can be confusing.